draft-ion.adoc

```diff
@@ -105,16 +120,13 @@ A single Ion instance resource without metadata is simply a standard JSON object
 }
 ----
 
-A single Ion instance resource with metadata is a JSON object that has a direct
-child `meta` name/value pair in addition to its other member name/value pairs:
+An Ion _Resource_ is a JSON object with a Metadata member named `meta` in addition to one or more other members (name/value pairs):
```

Comment:
Totally coming across in a positive light :)

I think I see your point here.  For me the distinction of a Resource vs a Link is due to reference/link expansion:

If I'm parsing a document and I see a JSON object with a `meta` member only, it *must* be a link (metadata *must* reflect other data external to itself, otherwise it cannot be metadata per the definition of the word).

If a JSON object has a `meta` member in addition to other members, then it is what I call 'materialized' (fully represented) and no GET request needs to be executed to obtain the data because it is known to already be in (local) in the document.

The spec (currently) uses the term 'Resource' to reflect that it is *not* a pointer, whereas a Link is a pointer.  

There should be a term for "this is an Ion Object that is not a Link" so can reference that concept succinctly.

Does this help?  I'm definitely open to feedback...